dotCMS/src/integration-test/java/com/dotcms/util/RelationshipUtilTest.java

```diff
@@ -0,0 +1,128 @@
+
+        final List<Contentlet> expectedResults = contentletAPI
+                .search(filter + " +languageId:" + languageID, 0, 0, null, user, false);
+
+        assertNotNull(testResults);
+        assertNotNull(expectedResults);
+        assertEquals(expectedResults.size(), testResults.size());
+        testResults.forEach(contentlet -> assertTrue(expectedResults.contains(contentlet)));
+
+    }
+
+    @Test
+    public void testFilterContentletWithIdentifier()
+            throws DotSecurityException, DotDataException {
+
+        final Contentlet contentlet = createContentlet();
+
+        try {
+            final String query = "+identifier:" + contentlet.getIdentifier() + " +languageId:" + languageID;
+            final List<Contentlet> testResults = RelationshipUtil
+                    .filterContentlet(languageID, contentlet.getIdentifier(), user);
+
+            final List<Contentlet> expectedResults = contentletAPI
+                    .search(query, 0, 0, null, user, false);
+
+            assertNotNull(testResults);
+            assertNotNull(expectedResults);
+            assertEquals(expectedResults.size(), testResults.size());
+            testResults.forEach(elem -> assertTrue(expectedResults.contains(elem)));
+        } finally {
+            if(contentlet != null && contentlet.getInode() != null){
+                ContentletDataGen.remove(contentlet);
+            }
+        }
+    }
+
+    @Test
+    public void testFilterContentletWithIdentifierAndLuceneQuery()
+            throws DotSecurityException, DotDataException {
+
+        final Contentlet contentlet = createContentlet();
+
+        try {
+            final String filter = "+contentType:Youtube";
+            final List<Contentlet> testResults = RelationshipUtil
+                    .filterContentlet(languageID,  contentlet.getIdentifier() + "," + filter , user);
+
+            final List<Contentlet> expectedResults = contentletAPI
+                    .search(filter + " +languageId:" + languageID, 0, 0, null, user, false);
+
+            assertNotNull(testResults);
+            assertNotNull(expectedResults);
+            assertEquals(expectedResults.size(), testResults.size());
+            testResults.forEach(elem -> assertTrue(expectedResults.contains(elem)));
+        } finally {
+            if (contentlet != null && contentlet.getInode() != null) {
+                ContentletDataGen.remove(contentlet);
+            }
+        }
+    }
+
+    private Contentlet createContentlet() throws DotDataException, DotSecurityException {
+        final ContentletDataGen contentletDataGen = new ContentletDataGen(
+                contentTypeAPI.find("Youtube").id());
+        Contentlet contentlet = contentletDataGen.next();
+
+        contentlet.setStringProperty("widgetTitle", "New YouTube Content");
+        contentlet.setStringProperty("url", "new-youtube-content");
+        contentlet = contentletDataGen.persist(contentlet);
+
+        return contentlet;
```

Comment:
Same as above, you can do this in one line

Reply:
it is not possible

Reply:
it was updated using `setProperty`  instead of `setStringProperty`